pkg/crc/cluster/status.go

```diff
@@ -24,7 +23,7 @@ func WaitForClusterStable(ctx context.Context, ocConfig oc.Config) error {
 	var count int // holds num of consecutive matches
 
 	for i := 0; i < retryCount; i++ {
-		status, err := GetClusterOperatorsStatus(ocConfig)
+		status, err := GetClusterOperatorsStatus(lister)
```

Comment:
Should this be passed `ctx` as `getStatus` needs a context too? (still not quite sure familiar with contexts :)

Reply:
yes added!